api/comments.rb

```diff
@@ -1,5 +1,8 @@
 get "#{APIPREFIX}/comments/:comment_id" do |comment_id|
-  comment.to_hash(recursive: bool_recursive).to_json
+  @comment = comment
+  comment_hash = @comment.to_hash(recursive: bool_recursive)
+  verify_or_fix_cached_comment_count(@comment, comment_hash)
```

Comment:
@robrap This same endpoint is being used when we retrieve response with all its comments. `recursive` parameter identifies whether to get children or not. So, if `recursive = true` then we already have the children and can compare with that. But if `recursive = false` then we need to make extra db call to get child count.